graphite_beacon/handlers/pagerduty.py

```diff
@@ -30,27 +31,47 @@ def notify(self, level, alert, value, target=None, ntype=None, rule=None):
         LOGGER.debug("Handler (%s) %s", self.name, level)
         message = self.get_short(level, alert, value, target=target, ntype=ntype, rule=rule)
         LOGGER.debug('message1:%s', message)
-        if level == 'normal':
-            event_type = 'resolve'
+
+        # Extract unique alert identifiers
+        alert_name = message[message.find("<")+1:message.find(">")]
```

Comment:
Can you use the context passed into the function for the alert name and metric name instead of pulling it out of the `message`?